_posts/2023-11-23-geospatial-techno.md

```diff
@@ -0,0 +1,76 @@
+---
+author: GeoSpatial Techno
+layout: post
+title: GeoServer installation methods on Windows
+date: 2023-11-23
+categories:   
+- Tutorials
+---
+
+[GeoSpatial Techno](https://www.youtube.com/@geospatialtechno) is a startup focused on geospatial information that is providing e-learning courses to enhance the knowledge of geospatial information users, students, and other startups. The main approach of this startup is providing quality, valid specialized training in the field of geospatial information.
+
+( [YouTube](https://www.youtube.com/@geospatialtechno)
+| [LinkedIn](https://www.linkedin.com/in/geospatialtechno)
+| [Facebook](https://www.facebook.com/geospatialtechno)
+| [Reddit](https://www.reddit.com/user/geospatialtechno)
+| [X](https://twitter.com/geospatialtechn)
+)
+
+----
+
+### GeoServer installation methods: “Windows Installer” and “Web Archive”
+GeoServer installation methods: “Windows Installer” and “Web Archive”
+In this session, we will talk about how to install GeoServer software by two commonly methods in Windows. If you want to access the complete tutorial, simply click on the [link](https://youtu.be/G47vV2od4yc).
+
+[![](https://img.youtube.com/vi/G47vV2od4yc/0.jpg)](https://www.youtube.com/watch?v=G47vV2od4yc)
+
+### Introduction
+GeoServer can be installed on different operating systems, since it's a Java based application. You can run it on any kind of operating system for which exists a Java virtual machine. GeoServer’s speed depends a lot on the chosen Java Runtime Environment (JRE). The latest versions of GeoServer are tested with both OracleJRE and OpenJDK. These versions are:
+- Java 17 for GeoServer 2.23 and above
+- Java 11 for GeoServer 2.15 and above
+- Java 8 for GeoServer 2.9 to GeoServer 2.22
+- Java 7 for GeoServer 2.6 to GeoServer 2.8
+- Java 6 for GeoServer 2.3 to GeoServer 2.5
+- Java 5 for GeoServer 2.2 and earlier
```

Comment:
It is not so useful to talk about old versions; we should stick with the currently supported versions only.
While some commercial providers can offer to support older environments - our community volunteers are already stretched to support a stable and maintainance branch :)

Reply:
You're both right. But users need to be aware that older versions are unsupported and won't receive fixes nor security updates, and contain well-known security vulnerabilities that have not been patched (so use at own risk).